cpu/stm32f1/include/cpu_conf.h

```diff
@@ -36,12 +37,113 @@ extern "C" {
 #endif
 
 /**
+ * @brief   Flash page configuration
+ * @{
+ */
+#define FLASHPAGE_SIZE      (2048U)
+
+#if defined(CPU_MODEL_STM32F103CB) || defined(CPU_MODEL_STM32F103RB)
+#define FLASHPAGE_NUMOF     (64U)
+#elif defined(CPU_MODEL_STM32F103RE)
+#define FLASHPAGE_NUMOF     (256U)
+#endif
+/** @} */
+
+/*
+ * @brief   Offset to reset handler on VTOR
+ */
+#define VTOR_RESET_HANDLER     0x4
+
+#if defined(CPU_MODEL_STM32F103RE)
+/*
+ * @brief   Flash partitioning for FW slots
+ * @{
+ */
+#define MAX_FW_SLOTS               (2)
+#define FW_SLOT_PAGES              (120)
+#define FW_SLOT_1                  (0x08004000)
+#define FW_SLOT_1_END              (0x0803FFFF)
+#define FW_SLOT_1_PAGE             (8)
```

Comment:
can some of these be calculated from others? Feels like there's redundant information, especially in combination with FLASHPAGE_SIZE.